docs/designers-developers/developers/tutorials/block-tutorial/creating-dynamic-blocks.md

```diff
@@ -145,7 +145,7 @@ There are a few things to notice:
 
 ## Live rendering in the block editor
 
-Gutenberg 2.8 added the [`<ServerSideRender>`](/packages/server-side-render/README.md) block which enables rendering to take place on the server using PHP rather than in JavaScript. 
+Gutenberg 2.8 added the [`<ServerSideRender>`](https://developer.wordpress.org/block-editor/packages/packages-server-side-render/) block which enables rendering to take place on the server using PHP rather than in JavaScript. 
```

Comment:
The first link is supposed to work, but it looks like the service that is converting links is not quite working as expected. We shouldn't be linking directly to the developer.wordpress.org domain, I think a more robust link might be to the Github repo source.

```suggestion
Gutenberg 2.8 added the [`<ServerSideRender>`](https://github.com/WordPress/gutenberg/tree/master/packages/server-side-render) block which enables rendering to take place on the server using PHP rather than in JavaScript. 
```

Reply:
Fair enough, as long as the link goes somewhere with the info!  :+1:

Reply:
...thinking about it though, would it not be best to keep someone in the documentation rather than linking out to readme files on the master branch which could have documentation targeted at at new code/features not yet in the current WordPress release?